front-end/src/components/general/NavBar.js

```diff
@@ -43,19 +51,46 @@ function NavBar() {
     <AppBar position="static">
       <CssBaseline />
       <Toolbar>
-        <Link to="/">
-          <Container>
-            <img alt="Travelo-Hey!" src="/placeholder_logo.jpg" />
-            <Typography variant="h4">
-              Travelo-Hey!
-            </Typography>
-          </Container>
-        </Link>
-        <div className="nav-bar-elements">
-          <Link to="/logOut">
-            Log out
-          </Link>
+        <div style={{ display: 'block', marginLeft: '0', marginRight: 'auto' }}>
+          <img src={logo} alt="Travelo-Hey!" width="200" height="114" />
         </div>
+        <nav style={{ marginLeft: 'auto', marginRight: '0' }}>
+          <ul style={{ listStyle: 'none', display: 'flex', flexDirection: 'row' }}>
+            <li>
+              <a
+                style={{ paddingRight: '50px', color: 'inherit', textDecoration: 'none' }}
```

Comment:
Since we're using React MUI, I'm curious why you chose to have an <a> tag instead of using MUI components like the <Link> tag just in terms of consistency with library and code throughout all files

Reply:
This is a great question. I was having difficult with the styling with the MUI components. Now that I have it working, I will work to update these to use MUI components. I'll update this by EOD today and re-request reviews. Thank you for bringing this forward.

Reply:
@mzuniga-1833381 -- I added the Link component instead of `<a>` HTML native tags. I also added MUI components for List. Are there other components I need to use? It doesn't seem like I need the Typography or Container components.

Reply:
From what I recall, I think that you can potentially get rid of the <nav> tags by doing `<List component="nav">` but 1) not sure if you already tried that and 2) I think it may come with some caveats so I'm not sure how that would affect your current code.

Reply:
Other than the `<nav>` tags, it all looks good!

Reply:
I'm having trouble finding any resources for the component='' tag. Do you have any resources I can take a look at?

Reply:
https://mui.com/material-ui/guides/composition/#how-does-it-work

Reply:
Skimming really quick this is where I found the component prop. It looks like there's two caveats so if you want to take a look at it if you have time but if everyone else is good with one minor aspect then I would be fine approving as I didn't find anything on not allowing `<nav>` tags for React MUI

Reply:
@mzuniga-1833381 - okay, I was able to incorporate the nav portion. The onMouse events work for the most part. It seems to get stuck if you go too fast over the links.